### ISSUE TITLE
Request: Alternate Artist Titles

### ISSUE THREAD

**Issue by DevinCampbell** [lidarr/Lidarr#804]
## Feature Request

I have an artist that is a Japanese band ( マキシマム ザ ホルモン ) that is usually called Maximum the Hormone in English. I'm not finding any downloads for it. Is there a way to make it find downloads for "Maximum the Hormone" as well? A way to specify alternate artist names would be great for alternate languages.

**Comment by Ether009** [comment]
It's the same for artists like Ayumi Hamasaki. Pretty much every non English band, that are officially selling outside of their home countries, have their titles in English on basically all sites. But, many sites does the mapping during search so a search gets the right albums, but Lidarr still can't parse it because it still doesn't understand the artist names.

**Comment by cjmanca** [comment]
Could anyone look into this? It has had an open pull request for 2 years that just needs a review. It prevents 90% of foreign language artist imports right now.

**Comment by webxI23** [comment]
Hi, I see I'm not alone with that problem. I have an artist with a french name that has a " ç " character in the artist name. When I rename the files based on the artist name, the file created cannot be played in an audio player ... The fact that the artist name cannot be edited/bypassed is problematic .....

**Comment by Fma15** [comment]
I would also love for ladarr to be able to search for alternative band names.

**Comment by barrelltitor** [comment]
Would be great to have this option as finding artists using their original language spelling might not get any results for quite a lot of languages,

**Comment by MrMxyzptlk** [comment]
Yup, same issue here; for example Modest Petrovich Mussorgsky is added to Lidarr by his Russian name Модест Петрович Мусоргский, so consequently none of his music can be found.

**Comment by davidfrickert** [comment]
Well it seems like this isn't going to be fixed anytime soon.

If anyone else is experiencing this issue and doesn't mind getting their hands slightly dirty, you can manually modify Lidarr's DB:

Open a terminal and change directory to Lidarr folder that contains the DB and start sqlite3 command line (install it if you don't have it).
```
(bash) $ cd <lidarr folder>
(bash) $ sqlite3 
sqlite> .open lidarr.db
```
Then just update data as needed:
```
sqlite> UPDATE Artists SET CleanName='Mariya Takeuchi',SortName='Mariya Takeuchi' WHERE CleanName='竹内まりや';
UPDATE ArtistMetadata SET Name='Mariya Takeuchi' WHERE Name = '竹内まりや';
```

You'll have to do this once per artist.

**Comment by apetresc** [comment]
That's a very intriguing suggestion @davidfrickert. Will this manual change get overridden the next time Lidarr choose to re-sync artist metadata? Or does this actually stick?

**Comment by davidfrickert** [comment]
Well I've just tried to refresh the metadata and no, it doesn't stick unfortunately, it gets overriden. meh.

**Comment by DevinCampbell** [comment]
It's been five years with no real development on this. If we can't get this, can we at least get translation in the import screen so we can go and find the albums ourselves and import them? Right now, I can't read the foreign characters to import the tracks.

**Comment by gmstrowabarton** [comment]
Really need this update with over 30+ artists in other languages in library and MusicBrainz has aliases and sort names in english available why is this almost 6 years now and no improvement.

**Comment by bakerboy448** [comment]
> Really need this update with over 30+ artists in other languages in library and MusicBrainz has aliases and sort names in english available why is this almost 6 years now and no improvement.



The primary development team is engaged in over five applications, including Lidarr, Prowlarr, Radarr, Readarr, Whisparr, and Sonarr. Currently, there is approximately one active developer contributing to these projects, with none of the contributors working on them full-time. Additionally, all members involved, whether in development or support roles, are volunteers and do not receive compensation for their contributions. Each team member balances their commitments alongside full-time jobs, family responsibilities, and other personal obligations.

As a result, the list of tasks to be completed across these projects, along with the various backends and modules, is extensive, while the available time for addressing these tasks is limited.
